optimum/exporters/onnx/base.py

```diff
@@ -731,7 +755,11 @@ def add_past_key_values(self, inputs_or_outputs: Dict[str, Dict[int, str]], dire
             inputs_or_outputs[f"{name}.{i}.decoder.key"] = {0: "batch_size", 2: decoder_sequence_name}
             inputs_or_outputs[f"{name}.{i}.decoder.value"] = {0: "batch_size", 2: decoder_sequence_name}
 
-            if direction == "inputs" or (self._behavior is ConfigBehavior.DECODER and self.use_past is False):
+            if (
+                direction == "inputs"
+                or (self._behavior is ConfigBehavior.DECODER and self.use_past is False)
+                or self.is_merged is True
```

Comment:
Small nit, but I would re-order those tests, for instance checking `self.is_merged` is True is much faster than `direction == "inputs"`.

Reply:
thanks!